src/compat/eq-131/ArkoudaZarrCompat.chpl

```diff
@@ -0,0 +1,411 @@
+module ArkoudaZarrCompat {
```

Comment:
Why do we have distinct modules for each one here? It seems they are all at least largely the same.

Also, we don't support 1.31 anymore.

Reply:
Zarr is only supported in Chapel as of 2.0, so the compatability module just includes the Zarr module body for the <2.0 versions. I'll remove the 1.31 version though.